external/fv3kube/fv3kube/config.py

```diff
@@ -19,7 +19,12 @@
 }
 
 TILE_COORDS_FILENAMES = range(1, 7)  # tile numbering in model output filenames
-RESTART_CATEGORIES = ["fv_core.res", "sfc_data", "fv_tracer.res", "fv_srf_wnd.res"]
+REQUIRED_RESTART_CATEGORIES = [
```

Comment:
Overall looks good to me...maybe would be nice to use a dataclass instead of a dict to get self-documenting/mypy/typechecking goodness:
@dataclass
```
class RestartNameConfig:
    core: str = "fv_core.res"
    surface: str = "sfc_data"
    tracer: str = "fv_tracer.res"
    surface_wind: str = "fv_srf_wnd.res"
```